admin/views/tabs/metas/paper-content/post-type-content.php

```diff
@@ -48,7 +50,7 @@
 		sprintf(
 			/* translators: %s expands to the post type's name. */
 			__( 'the archive for %s', 'wordpress-seo' ),
-			$plural_label
+			$label
```

Comment:
It should be escaped on same way as previous location regardless to original code.

Please change comments for translators according to new changes.